### PR TITLE
Upgrade sbt to 1.4.9

### PR DESCRIPTION
**Pull request by johanandren**
Plus custom launcher script for travis

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -9,7 +9,16 @@ scala:
...
+install: 
+  - jabba install "adopt@~1.$TRAVIS_JDK.0-0" && jabba use "$_" && java -Xmx32m -version
+  - |
+    # update this only when sbt-the-bash-script needs to be updated
+    export SBT_LAUNCHER=1.4.9
+    export SBT_OPTS="-Dfile.encoding=UTF-8"
+    curl -L --silent "https://github.com/sbt/sbt/releases/download/v$SBT_LAUNCHER/sbt-$SBT_LAUNCHER.tgz" > $HOME/sbt.tgz
+    tar zxf $HOME/sbt.tgz -C $HOME
+    sudo rm /usr/local/bin/sbt
+    sudo ln -s $HOME/sbt/bin/sbt /usr/local/bin/sbt
```
Comment (johanandren) [inline .travis.yml:21]: Can't use travis provided launcher script for 1.4.8+ (for now) since moved to maven central now

File: .travis.yml [hunk 2]
```diff
@@ -28,7 +37,7 @@ cache:
     - $HOME/.jabba/jdk
 
 # script for the default 'test' stage:
-script: sbt -jvm-opts .jvmopts-travis -Dakka.build.scalaVersion=$TRAVIS_SCALA_VERSION ";update ;mimaReportBinaryIssues ;test:compile ;validateCompile ;headerCheckAll"
+script: cp .jvmopts-travis .jvmopts && sbt -Dakka.build.scalaVersion=$TRAVIS_SCALA_VERSION ";update ;mimaReportBinaryIssues ;test:compile ;validateCompile ;headerCheckAll"
```
Comment (johanandren) [inline .travis.yml:40]: `-jvm-opts` was custom for travis launcher (or something)

**Review by octonato — APPROVED** [review]
👍

**Comment by raboof** [comment]
master is on 1.5.2 now

**Pull request closed without merging**